src/OpenTelemetry/Metrics/Aggregators/Aggregator.cs

```diff
@@ -25,6 +27,32 @@ namespace OpenTelemetry.Metrics.Aggregators
     public abstract class Aggregator<T>
         where T : struct
     {
+        private long startTime;
```

Comment:
what if we create this as DateTime instead of long? Why: every time that we use the agreggator we will keep doing datetime -> binary or binary -> datetime. If we keep that as datetime, we would prevent those conversions and use as long when we need. What do you think?

Reply:
I did this originally as a `DateTime` and would indeed prefer to do it that way, but I ended up going with `long` so that I could use the atomic assigment via `Interlocked.Exchange`. DateTime is not supported in that operation.

Reply:
just add a comment of the time units of this field

Reply:
I'd suggest to use Ticks instead of binary.